app/cli/cmd/workflow_workflow_run_list.go

```diff
@@ -23,14 +23,15 @@ import (
 	"github.com/chainloop-dev/chainloop/app/cli/internal/action"
 	"github.com/jedib0t/go-pretty/v6/table"
 	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
```

Comment:
I'd rather that you implement this functionality. 

I get uneasy with the warning in that package

```
Warning: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.
```

Reply:
Sure, will do.